west.yml

```diff
@@ -190,7 +190,7 @@ manifest:
       revision: cfd050ff38a9d028dc211690b2ec35971128e45e
       path: modules/lib/open-amp
     - name: openthread
-      revision: b7aa7686d3f4e326bd25abc234b209159977c26d
+      revision: 41bc49da49736fbdfdfa231f7b2311f29dcc4979
```

Comment:
Hi, any reason why to downgrade OpenThread here?

Reply:
@edmont, no reason at all. Missing rebase. Thanks for notice.

Reply:
Fixed!